lib/OrdinaryDiffEqIMEXMultistep/src/algorithms.jl

```diff
@@ -21,6 +25,10 @@ function CNAB2(; chunk_size = Val{0}(), autodiff = Val{true}(), standardtag = Va
         extrapolant)
 end
 
+@doc generic_solver_docstring("TBD",
+    "CNLF2",
+    "IMEX Multistep method.",
+    "REF TBD", "", "")
```

Comment:
CNAB2 and CNLF2 description and reference @ChrisRackauckas.

Reply:
Crank-Nicholson Adams-Bashforth 2, and Crank-Nicholson Leapfrong 2. For the first, https://global-sci.org/intro/article_detail/ijnamB/229.html and https://onlinelibrary.wiley.com/doi/epdf/10.1002/fld.2035. https://www.sciencedirect.com/science/article/abs/pii/S0893965920302445 and https://www.sciencedirect.com/science/article/pii/S0377042714004336 for the other.